client/my-sites/post-share/style.scss

```diff
@@ -0,0 +1,140 @@
+.posts__list .post-share__wrapper {
```

Comment:
Can we remove the nesting here? Do we need to write `.posts__list .post-share__wrapper` instead of just `.post-share__wrapper`?